recipes/pyston_lite/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "pyston_lite" %}
+{% set pyston_version = "2.3.4" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ pyston_version }}
+
+source:
+  - url: https://github.com/pyston/pyston/archive/pyston_{{ pyston_version }}.tar.gz
+    sha256: 9989ce0f3a83611b11f79121d29ceec84c3cb4b4a8c3de4f8d97eb42042f4cee
+  # pyston-lite uses a preprocessing script from inside the LuaJIT repository (that is not
+  # packaged as part of the luajit package), but only as a build tool (ie is not included
+  # in the built artifacts).
+  - url: https://github.com/LuaJIT/LuaJIT/archive/e2c312e0deb874aa5fa8ce502c08d87deb38e82f.tar.gz
+    sha256: bff4764c19466def90667d2f7fcbd0781b8e62476b12105868f8e44babd03f8e
+    folder: pyston/LuaJIT/
+
+build:
+  number: 0
+
+  skip: True  # [not linux]
+  skip: True  # [py!=38]
+
+# In theory the requirements can/should be specified per-output, but in practice
+# conda-build will produce a corrupt package if the compiler dependency is not
+# additionally specified at this top scope. It is required to declare the dependency
+# both here and in the outputs: section as well; removing either will cause the c
+# compiler to not be installed.
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
The top-level package can be one of the outputs, so you can do something like this to avoid redundant requirements sections:

```yaml

package:
  name: pyston_lite

# define pyston_lite metadata here because it is the main recipe

outputs:
  - name: pyston_lite  # just name as output, but it's already defined above
  - name: pyston_lite_autoload
    build:
      noarch: python
    # complete secondary output information

# all outputs get the same about section
about:

```

Reply:
No, please have the top-level package and outputs separate. This has bitten me multiple times. It works but only in very specific circumstances.